src/trade/TradeCard.js

```diff
@@ -169,7 +151,9 @@ export default class TradeCard extends Component {
             serverContractModelToChartContractModel(lastBoughtContract);
 
         // contract error is not tied to trade, but symbol, thus not in tradeErrors
-        const tradeError = (propsContract ? propsContract.error : undefined) || errorToShow(tradeErrors);
+        const tradeError = propsContract ?
+            Object.assign(tradeErrors, { contractError: propsContract.error }) :
```

Comment:
In this case, I think you are right, since we do not yet really care about the type of error. Will change